platform/infra/lib/infra/vpc.ts

```diff
@@ -9,8 +9,42 @@ export default class VpcStack extends Stack {
     constructor(scope: Construct, id: string, props: StackProps) {
         super(scope, id, props);
         this.vpc = new Vpc(this, "appmod-vpc", {
-            vpcName: "appmod-vpc"
+            vpcName: "appmod-vpc",
+            maxAzs: 3,
+            ipAddresses: IpAddresses.cidr('192.168.0.0/16'),
+            subnetConfiguration: [
```

Comment:
so based on maxAzs it will result in three public and three private?

Reply:
Yes. Each AZ by default gets a public and private subnet, or whatever is in subnet configuration.